controller/instance_manager_controller.go

```diff
@@ -181,6 +186,17 @@ func NewInstanceManagerController(
 		},
 	})
 
+	kubeNodeInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		UpdateFunc: func(oldObj, cur interface{}) {
+			curNode := cur.(*v1.Node)
+			imc.enqueueKubernetesNode(curNode)
+		},
+		DeleteFunc: func(obj interface{}) {
+			node := obj.(*v1.Node)
+			imc.enqueueKubernetesNode(node)
```

Comment:
I noticed that the InstanceManager is slow to react when node changes status (schedulable/unschedulable).

Therefore, I added to the kubeNodeInformer so that InstanceManager react faster.